app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::ReviewsController < ApplicationController
+  def index
+    render json: Review.where(coffeeshop_id: params["coffeeshop_id"])
+  end
```

Comment:
ideally you grab the coffee shop reviews when grabbing the coffee_shop#show using AR associations